lib/ansible/plugins/connection/ssh.py

```diff
@@ -296,7 +296,9 @@
             version_added: '2.7'
       sftp_batch_mode:
         default: true
-        description: 'TODO: write it'
+        description: '
```

Comment:
Are these quotes intentional? Should it be a list of string instead?

Also, should we explain what the batch mode does?

Reply:
the original was using quotes to avoid the YAML issue with `: ` in the TODO